environments/staging/group_vars/all/robots_txt.yml

```diff
@@ -14,3 +14,9 @@ archive_robots_txt:
     - {name: 'ARCHIVE TESTING', crawl_delay: 10}
   disallow_list:
     - /
+vendor_robots_txt:
+  user_agents:
+    - {name: '*'}
+    - {name: 'ARCHIVE TESTING', crawl_delay: 10}
+  allow_list:
+    - /
```

Comment:
In [this comment](https://github.com/openstax/cnx/issues/744#issuecomment-546407689), Phil mentioned to use `disallow /` rather than `allow /`. Is this the relevant piece of code here?

Reply:
I see that there is a disallow below. What's the difference between this one and the one at roles/robots_txt/defaults/main.yml

Reply:
Fixed.